picturae_csv_create.py

```diff
@@ -426,17 +395,20 @@ def csv_colnames(self):
         # self.logger.info("merged csv written")
 
 
-
-    def flag_missing_data(self):
-
+    def missing_data_masks(self):
```

Comment:
split the missing data masks , into two functions, one to create the masks, the second to check them and raise an error.